docs/samples/modelmesh_sdk_sample.ipynb

```diff
@@ -0,0 +1,390 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample on how to use modelmesh-serving with SDK v1beta1 python\n",
+    "\n",
+    "## This notebook shows how to use the python sdk with ModelMesh to create an InferenceService"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kubernetes import client\n",
+    "from kserve import constants\n",
+    "from kserve import V1beta1InferenceService\n",
+    "from kserve import V1beta1InferenceServiceSpec\n",
+    "from kserve import V1beta1PredictorSpec\n",
+    "from kserve import V1beta1SKLearnSpec\n",
+    "from kserve import V1beta1TFServingSpec\n",
+    "from kserve import V1beta1StorageSpec\n",
+    "from kserve import KServeClient"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the global variables to be used in InferenceService creation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "namespace = 'modelmesh-serving'\n",
+    "name='msnist-sample'\n",
+    "protocol_version='v2'\n",
+    "api_version = constants.KSERVE_V1BETA1"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define InferenceService with the proper ModelMesh annotation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "isvc = V1beta1InferenceService(\n",
+    "    api_version=constants.KSERVE_V1BETA1,\n",
```

Comment:
Looks like you're using `constants.KSERVE_V1BETA1` here and not the variable defined above, probably just remove the global variable above